echo-notifications/echo-notifications.gradle

```diff
@@ -36,3 +37,14 @@ dependencies {
     testImplementation "com.icegreen:greenmail:1.4.0"
     testImplementation "org.apache.httpcomponents:httpclient"
 }
+
+sourceSets {
+  main {
+    java {
+      srcDirs = [] // don't compile Java code twice
+    }
+    groovy {
+      srcDirs = [ 'src/main/groovy', 'src/main/java' ]
```

Comment:
This was required due to introduction of the new Java sources in `echo-notification`, which depend on the Groovy classes.